components/advocates/AdvocateCard.vue

```diff
@@ -1,79 +1,66 @@
 <template>
-  <article class="advocate-card">
-    <div
-      class="advocate-card__picture"
-      :style="`background-image: ${decorate(image)};`"
-    />
-    <div class="advocate-card__copy">
-      <h3 class="advocate-card__name">
-        {{ name }}
-      </h3>
+  <AppCard
+    class="advocate-card"
+    :image="image"
+    :title="name"
+    :tags="formattedRegion"
+  >
+    <div>
       <p class="advocate-card__location">
+        <Map20 class="advocate-card__icon" />
         {{ location }}
       </p>
-      <h3 class="advocate-card__focus-areas-title">
-        Focus areas
-      </h3>
-      <p class="advocate-card__focus-areas">
-        {{ areas }}
+      <p class="advocate-card__contact">
+        <LogoSlack20 class="advocate-card__icon" />
+        {{ slackId }}
```

Comment:
As the first approach, we can use [this method](https://slack.com/intl/en-es/help/articles/360003827751-Create-a-link-to-a-members-profile-#free,-standard,-and-plus-plans-1) to open a conversation with the advocates. Could you prepare the `AppLink` here for that instead of having just the `{{ slackId }}`?